variables_iam_static_users.tf

```diff
@@ -0,0 +1,32 @@
+variable "iam_user_s3_full" {
+  description = "Enable IAM user(s) with S3 bucket full access"
+  default     = false
+}
+
+variable "iam_user_s3_full_names" {
+  type        = "list"
+  description = "Names of the IAM users with S3 bucket full access"
+  default     = []
+}
+
+variable "iam_user_s3_list_delete" {
+  description = "Enable IAM user(s) with S3 bucket list/delete permissions"
+  default     = false
```

Comment:
Maybe this can be checked if the list is empty or not.

Reply:
todo: check if list has 0 elements instead